src/download/exchangeDirectoryParser.ts

```diff
@@ -20,38 +20,48 @@ function getFiles(directory): fs.Dirent[] {
  * @export
  * @param {string} directory Directory we want to download to
  * @param {boolean} [removeFiles=true] Whether to remove the zip files after extraction
- * @returns {Promise<void>} Just a promise to indicate we are done.
+ * @returns {Promise<string>} The extracted file's path
+ */
+export function extractFile(file: string, removeFiles = true): Promise<string> {
+  return new Promise((resolve, reject) => {
+    const filepath = path.join(path.dirname(file), path.basename(file, ".zip"));
```

Comment:
```suggestion
    const filepath = file.slice(0, -4); // Strip .zip
```
```suggestion
    const filepath = file.replace(/\.zip$/, "");
```
The suggestions are equivalent when called from `extractFiles`, but the second is safer when called from elsewhere. Both avoid the overhead of using the path methods.

Reply:
I don't think scale is an issue so Id rather use the path methods for now.